src/script/run-cbt.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh -x
+
+usage() {
+    local prog_name=$1
+    shift
+    echo <<EOF
+$prog_name [options] <config-file>...
+
+options:
+  -a,--archive-dir    directory in which the test result is stored
+  -h,--help           print this help message
+
+example:
+  $prog_name -a /tmp radosbench-write.yaml
+EOF
+}
+
+MDS=0 MGR=1 OSD=3 MON=1 ../src/vstart.sh -n --without-dashboard --memstore -X \
+   -o "memstore_device_bytes=34359738368" \
+   --crimson --nodaemon --redirect-output \
+   --osd-args "--memory 4G --smp 1 --cpuset 0"
```

Comment:
In the matter of `--cpuset 0`: aren't we overriding the behavior introduced to `vstart,sh` by the first commit?

Reply:
yeah, i failed to update the driver script. will remove it in the next revision.